dune-project

```diff
@@ -56,15 +56,16 @@
   cohttp-lwt-unix
   (elastic-apm
    (= :version))
-  (elastic-apm-async-reporter
+  (elastic-apm-lwt-reporter
    (= :version))))
 
 (package
  (name elastic-apm-async-reporter)
  (depends
   (ocaml
    (>= 4.12.0))
-  async
+  (async
+   (< v0.15.0))
```

Comment:
this is temporary. I'll propose a patch to blue_http later to get it working with core 0.15.0